test/helper/utils.js

```diff
@@ -0,0 +1,10 @@
+const chai = require('chai');
```

Comment:
'chai' is assigned a value but never used
'app' is assigned a value but never used